src/universal_agent/tgtg/cli.py

```
@@ -0,0 +1,744 @@
+    """Run a region scan to populate / refresh the item catalog."""
+    lat = args.lat or TGTG_LATITUDE
+    lon = args.lon or TGTG_LONGITUDE
+    radius = args.radius or TGTG_RADIUS
+
+    print(f"Scanning area: lat={lat}, lon={lon}, radius={radius}km …")
+    client = build_client(email=TGTG_EMAIL or None)
+    report = scan_and_store(client, lat, lon, radius)
+    print(report)
+
+    if report.new_item_ids:
+        print(f"\n🆕 New stores found:")
```

Ruff (0.15.2)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)